tests/common.go

```diff
@@ -1827,12 +1828,18 @@ func PerformSystemCheck() {
 					TimeBeforeRetry: 10 * time.Second,
 				})
 				if len(file) != 0 || err != nil {
-					log.Errorf("an error occurred, collecting bundle")
-					CollectSupport()
-					dash.VerifySafely(file == "", true, fmt.Sprintf("Core generated on node %s, Core Path: %s", n.Name, file))
+					dash.Errorf("Core generated on node %s, Core Path: %s", n.Name, file)
+					// Collect Support Bundle only once
+					if !foundCore {
+						CollectSupport()
+					}
+					foundCore = true
```

Comment:
you don't need to continue if core file was found. Just `break` from the loop as the `CollectSupport()` method has already collected all needed logs and diags from all nodes

Reply:
I am continuing the loop to logs message if there is core in other nodes too. it wont collect support bundle but just logs the info if core found. Let me know if this doesn't make sense